firefox_desktop/views/desktop_daily_retention.view.lkml

```diff
@@ -0,0 +1,355 @@
+include: "//looker-hub/firefox_desktop/views/*"
+
+view: +desktop_cohort_daily_retention {
+
+  # Hide metric columns showing as dimensions
+  dimension: cohort {
+    hidden: yes
+    sql: ${TABLE}.cohort ;;
```

Comment:
you don't need the `sql` portion, that should already be present for these dimensions